runtime/strings/bytes.cpp

```diff
@@ -122,10 +122,22 @@ SortBytes hook_BYTES_substr(SortBytes input, SortInt start, SortInt end) {
   uint64_t ustart = get_ui(start);
   uint64_t uend = get_ui(end);
   if (uend < ustart) {
-    throw std::invalid_argument("Invalid string slice");
+    throw std::invalid_argument(
+        std::string("Invalid string slice for string \"")
+        + std::string(input->data) + std::string("\": Requested start index ")
```

Comment:
I know we're trying to keep the code simple, but what if the string data is huge? Won't this produce a potentially enormous error message?

Reply:
That's fair, I removed the string from the error message. It should still be more informative than before.